wp-content/themes/core/blocks/tribe/terms/render.php

```diff
@@ -16,7 +16,16 @@
 echo '<div ' .  wp_kses_data( get_block_wrapper_attributes() ) . '>';
 
 if ( 0 === count( $terms_block_terms ) ) {
-	echo '<!-- Terms block: No terms to list. -->';
+	if ( strpos( $_SERVER['REQUEST_URI'], 'wp-admin' ) !== false || strpos( $_SERVER['REQUEST_URI'], 'wp-json' ) !== false ) {
+		echo '<ul class="wp-block-tribe-terms__list">';
+		echo '<li class="wp-block-tribe-terms__term">';
+		echo '<span class="wp-block-tribe-terms__link t-category">'. esc_html__( 'Terms Display Here', 'tribe' ) .'</span>';
+		echo '</li>';
+		echo '</ul>';
+	} else {
+		echo '<!-- Terms block: No terms to list. -->';
+	}
+
```

Comment:
Curious: Do we _need_ to render something different in the editor than the admin? I wonder if we render just don't render anything, always. If it needs to be visually distinguished in the editor, maybe we add an empty results css class to the container and use editor-specific css?

Reply:
Good call - I ended up taking this route in my newest commit. I'm checking if the block is empty in the editor and adding a pseudo element styled how it would look in the FE. I like this approach more, it's just not possible to translate the string (which I don't know is a problem, either). Let me know how you like this approach vs Chris' separate PR.